runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatChanges.java

```diff
@@ -106,13 +107,37 @@ private void init()
+		// Many of these retrieved from https://oldschool.runescape.wiki/w/User:Choppe/Sandbox2
 		add(combo(food(11), dec(ATTACK, 2)), JUG_OF_WINE);
 		add(combo(food(14), dec(ATTACK, 3)), BOTTLE_OF_WINE);
 		add(combo(2, food(5), boost(STRENGTH, 6), heal(ATTACK, -4)), PREMADE_WIZ_BLZD, WIZARD_BLIZZARD);
 		add(combo(2, food(5), boost(STRENGTH, 4), heal(ATTACK, -3)), PREMADE_SGG, SHORT_GREEN_GUY);
 		add(combo(2, food(5), boost(STRENGTH, 7), heal(ATTACK, -4)), PREMADE_DR_DRAGON, DRUNK_DRAGON);
 		add(combo(2, food(5), boost(STRENGTH, 7), heal(ATTACK, -4)), PREMADE_CHOC_SDY, CHOC_SATURDAY);
+		add(combo(2, food(5), new SimpleStatBoost(STRENGTH, true, perc(0.05, 1)), new BoostedStatBoost(ATTACK, false, perc(0.02, -3))), BRANDY);
+		add(combo(2, food(5), new SimpleStatBoost(STRENGTH, true, perc(0.05, 1)), new BoostedStatBoost(ATTACK, false, perc(0.02, -3))), GIN);
+		add(combo(2, food(5), new SimpleStatBoost(STRENGTH, true, perc(0.04, 1)), new BoostedStatBoost(ATTACK, false, perc(0.05, -3))), GROG);
+		add(combo(2, food(5), new SimpleStatBoost(STRENGTH, true, perc(0.05, 1)), new BoostedStatBoost(ATTACK, false, perc(0.02, -3))), VODKA);
+		add(combo(2, food(5), new SimpleStatBoost(STRENGTH, true, perc(0.05, 1)), new BoostedStatBoost(ATTACK, false, perc(0.02, -3))), WHISKY);
+		add(combo(2, food(1), boost(STRENGTH, perc(.02, 1)), new BoostedStatBoost(ATTACK, false, perc(.06, -1))), BEER);
 		add(combo(4, boost(ATTACK, 5), boost(STRENGTH, 5), heal(MAGIC, -5), heal(PRAYER, -5)), BLOOD_PINT);
+		add(combo(2, food(1), boost(STRENGTH, 2), new BoostedStatBoost(ATTACK, false, perc(0.05, -2))), ASGARNIAN_ALE);
+		add(combo(5, food(1), boost(STRENGTH, 3), new BoostedStatBoost(ATTACK, false, perc(0.05, -3))), ASGARNIAN_ALEM);
+		add(combo(3, food(1), boost(WOODCUTTING, 1), new BoostedStatBoost(ATTACK, false, perc(0.02, -2)), new BoostedStatBoost(STRENGTH, false, perc(0.02, -2))), AXEMANS_FOLLY);
+		add(combo(3, food(2), boost(WOODCUTTING, 2), new BoostedStatBoost(ATTACK, false, perc(0.02, -2)), new BoostedStatBoost(STRENGTH, false, perc(0.02, -2))), AXEMANS_FOLLYM);
+		add(combo(4, food(1), boost(THIEVING, 1), boost(ATTACK, 1),  new BoostedStatBoost(DEFENCE, false, perc(0.06, -3)), new BoostedStatBoost(STRENGTH, false, perc(0.06, -3))), BANDITS_BREW);
+		add(combo(3, food(1), new SimpleStatBoost(COOKING, true, perc(0.05, 1)), new BoostedStatBoost(ATTACK, false, perc(0.05, -2)), new BoostedStatBoost(STRENGTH, false, perc(0.05, -2))), CHEFS_DELIGHT);
+		add(combo(3, food(2), new SimpleStatBoost(COOKING, true, perc(0.05, 2)), new BoostedStatBoost(ATTACK, false, perc(0.05, -3)), new BoostedStatBoost(STRENGTH, false, perc(0.05, -3))), CHEFS_DELIGHTM);
+		add(combo(3, food(1), boost(FARMING, 1), new BoostedStatBoost(ATTACK, false, perc(0.02, -2)), new BoostedStatBoost(STRENGTH, false, perc(0.02, -2))), CIDER);
+		add(combo(3, food(2), boost(FARMING, 2), new BoostedStatBoost(ATTACK, false, perc(0.02, -3)), new BoostedStatBoost(STRENGTH, false, perc(0.02, -3))), CIDERM1);
```

Comment:
Instead of `CIDERM1` this should be `MATURE_CIDER`

Reply:
Fixed